src/javascript/app/pages/user/reset_password.js

```diff
@@ -49,7 +50,10 @@ const ResetPassword = (() => {
                 localized_message: localize('You have a new binary password.'),
                 localized_title  : localize('Binary password'),
                 ok_text          : localize('Done'),
-                onConfirm        : () => Login.redirectToLogin(true),
+                onConfirm        : () => {
+                    Login.redirectToLogin(true);
+                    Client.sendLogoutRequest(true);
```

Comment:
Do we still need `Login.redirectToLogin` above? Because `sendLogoutRequest` can take a redirectTo url as a second parameter, so we can pass it there.

Reply:
Hi @akmal,
Thanks for bringing it up. It seems on passing **true**   to `Client.sendLogoutRequest();` , redirect to login page is done inside the code.  So looks like its good to go with **Client.sendLogoutRequest(true);. Could you please verify?